FastReport.Base/Utils/Config.cs

```diff
@@ -109,8 +110,18 @@ public static bool PreparedCompressed
         /// </summary>
         public static string ApplicationFolder
         {
-            get { return AppDomain.CurrentDomain.BaseDirectory; }
+            get { return (string.IsNullOrWhiteSpace(customApplicationFolder) ? AppDomain.CurrentDomain.BaseDirectory : customApplicationFolder); }
         }
+        
+
+        /// <summary>
+        /// Gets and sets the custom application folder.
+        /// </summary>
+        public static string CustomApplicationFolder
+        {
+            get { return customApplicationFolder; }
+            set { customApplicationFolder = value; }
```

Comment:
I don't think you should add a separate property to set up the `customApplicationFolder` field. It might be worth just adding a setter for the `ApplicationFolder` that sets the value of the `customApplicationFolder` field.

Reply:
that does not work. you read directly from AppDomain.CurrentDomain.BaseDirectory

public static string ApplicationFolder
{
    get { return AppDomain.CurrentDomain.BaseDirectory; }
}

Reply:
You can keep the static field, but not add a separate property, add a setter in an `ApplicationFolder` property

Reply:
ok i understand